README.md

```diff
@@ -38,7 +38,7 @@ var App = React.createClass({
 
 ## All Props
 
-Please refer to the [official Plaid Link docs](https://plaid.com/link/) for
+Please refer to the [official Plaid Link docs](https://plaid.com/docs/link/) for
```

Comment:
It works when I copy and paste it or click it in [my fork's README](https://github.com/Subzidion/react-plaid-link/#all-props)